reference/5.1/Microsoft.PowerShell.Core/About/about_PSItem.md

```diff
@@ -0,0 +1,230 @@
+Locale: en-US
+ms.date: 12/05/2022
+online version: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_psitem?view=powershell-7.3&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about PSItem
+---
+# about_PSItem
+
+## Short description
+
+The automatic variable that contains the current object in the pipeline object.
+
+## Long description
+
+PowerShell includes the `$PSItem` variable and its alias, `$_`, as
+[automatic variables][01] in scriptblocks that process pipeline objects.
+
+You can use this variable in commands that perform an action on every object or
+on selected objects in a pipeline.
+
+There are a few common use cases for `$PSItem`:
+
+- in the **Process** parameter of the `ForEach-Object`cmdlet
+- in the **FilterScript** parameter of the `Where-Object` cmdlet
+- in the intrinsic methods **ForEach** and **Where**
+- with delay-bind scriptblock parameters
+- in a `switch` statement's conditional scriptblocks
+- in the `process` block of a function
+- in a `filter` definition
+- in the substitution operand scriptblock of the `-replace` operator
+
+The rest of this article includes examples of using `$PSItem` for these use
+cases.
+
+## ForEach-Object Process
+
+The [ForEach-Object][02] cmdlet is designed to operate
+on objects in the pipeline, executing the **Process** parameter's scriptblock
+once for every object in the pipeline.
+
+You can use `$PSItem` in the **Process** parameter's scriptblock but not in the
+**Begin** or **End** parameter scriptblocks. If you reference `$PSItem` in the
+**Begin** or **End** parameter scriptblocks, the value is `$null` because those
+scriptblocks don't operate on each object in the pipeline.
+
+```powershell
+$parameters = @{
+    Begin   = { Write-Host "PSItem in Begin is: $PSItem" }
+    Process = {
+        Write-Host "PSItem in Process is: $PSItem"
+        $PSItem + 1
+    }
+    End     = { Write-Host "PSItem in End is: $PSItem" }
+}
+
+$result = 1, 2, 3 | ForEach-Object @parameters
+
+Write-Host "Result is: $result"
+```
+
+```output
+PSItem in Begin is:
+PSItem in Process is: 1
+PSItem in Process is: 2
+PSItem in Process is: 3
+PSItem in End is:
+Result is: 2 3 4
+```
+
+## Where-Object FilterScript
+
+The [Where-Object][03] cmdlet is designed to filter objects in the pipeline.
+
+You can use `$PSItem` in the **FilterScript** parameter, which executes once
+for each input object in the pipeline.
+
+```powershell
+1, 2, 3 | Where-Object -FilterScript { ($PSItem % 2) -eq $true }
+```
+
+```output
+2
+```
+
+The **FilterScript** checks to see if the current object is even and filters
+out any odd values, returning only `2` from the original list.
+
+## ForEach and Where methods
+
+Both the [ForEach][04] and [Where][05] intrinsic methods for arrays take a
+scriptblock as an input parameter. You can use the `$PSItem` in those
+scriptblocks to access the current object.
+
+```powershell
+@(1, 2, 3).ForEach({ $PSItem + 1 }).Where({ ($PSItem % 2) -eq $true })
```

Comment:
This example is confusing. Is the value `3` the incremented value or the original value of `$PSItem`? Can we come up with a better example?